widgetastic_manageiq/__init__.py

```diff
@@ -3660,7 +3660,31 @@ def fill(self, value):
         return True
 
 
-class PotentiallyInvisibleTab(Tab):
+class WaitTab(Tab):
+    def child_widget_accessed(self, widget):
+        _to_try = 2
+        _tries = 0
+        while _tries < _to_try:
+            _tries += 1
+            self.select()
+            try:
+                wait_for(self.is_active, delay=.1, num_sec=2)
+            except TimedOutError:
+                self.logger.exception(
```

Comment:
warn ?

Reply:
could, exception log gives us the exception as well as the passed message.